pyensembl/genome.py

```diff
@@ -169,17 +169,19 @@ def _get_gtf_path(self, download_if_missing=False, overwrite=False):
             download_if_missing=download_if_missing,
             overwrite=overwrite)
 
-    def _get_transcript_fasta_path(
+    def _get_transcript_fasta_paths(
             self,
             download_if_missing=False,
             overwrite=False):
         if not self.has_transcript_fasta:
             raise ValueError("No transcript FASTA source for %s" % self)
-        return self._get_cached_path(
-            field_name="transcript-fasta",
-            path_or_url=self._transcript_fasta_path_or_url,
-            download_if_missing=download_if_missing,
-            overwrite=overwrite)
+        return [
+            self._get_cached_path(
+                field_name="transcript-fasta",
+                path_or_url=path,
+                download_if_missing=download_if_missing,
+                overwrite=overwrite
+            ) for path in self._transcript_fasta_paths_or_urls]
```

Comment:
Really small syntax nit but I think it's more idiomatic to move the closing paren to the previous line.